tasks/update-ocp-tag/README.md

```diff
@@ -8,10 +8,13 @@ Tekton task to update version tag of FBC pull-spec
 
 | Name | Description | Optional | Default value |
 |------|-------------|----------|---------------|
-| fromIndex | The source Index image (catalog of catalogs) FBC fragment | No | - |
-| targetIndex | Index image (catalog of catalogs) the FBC fragment will be added to | No | - |
-| binaryImage | OCP binary image to be baked into the index image | No | - |
+| dataPath | Path to the JSON string of the merged data to use in the data workspace | No | data.json |
```

Comment:
If it has a default value, it should be optional

Reply:
Fixed, I always miss this. Thanks!